tasks/configure-pipeline-local-fs.yml

```diff
@@ -0,0 +1,140 @@
+---
```

Comment:
I think it's worth commeting that "pipeline local filesystem" is an strange way to call an ssh-based remote space, so adding some kind of clarification would be intersting.

Reply:
Agree!! 👍